roles/openstack-setup/tasks/users.yml

```diff
@@ -15,8 +15,8 @@
                     login_password={{ secrets.admin_password }}
                     insecure={{ insecure|default(omit) }}
   with_items: "{{ keystone.services }}"
-  when: 
-    - (item.name.enabled is undefined or item.name.enabled|bool)
+  when:
+    - ({{item.name}}.enabled is undefined or {{item.name}}.enabled|bool)
```

Comment:
I don't see the difference. For all the cases, they get the same result
case1: item.name.enabled is undefined
case2: item.name.enabled is true
case3: item.name.enabled is false